chromadb/utils/embedding_functions/onnx_mini_lm_l6_v2.py

```diff
@@ -97,7 +97,7 @@ def _download(self, url: str, fname: str, chunk_size: int = 1024) -> None:
         which makes mypy unhappy. If some smart folk knows how to fix this in an
         elegant way, please do so.
         """
-        with httpx.stream("GET", url) as resp:
+        with httpx.stream("GET", url, timeout=None) as resp:
```

Comment:
I don't think we'd want to unilaterally set timeout to None. The default is 5 seconds - https://www.python-httpx.org/advanced/timeouts/. 

Could we instead make this a parameter at EF construction time that one can set?

Reply:
Hi @HammadB 
fully agree that None does not make sense. 

I think the best would be to offer some **configuration value** or **environment variable** which can be set matching the timings of the infrastructure (as this might vary between users). The best would be if httpx offers this out of the box.

In our case there was some scanning of images which happens during the download so the initial download takes some time (to be secure) and later the stuff is cached and immediately available.